pkgs/development/libraries/openfec/default.nix

```diff
@@ -0,0 +1,44 @@
+{ stdenv, lib, fetchzip, cmake }:
+
+stdenv.mkDerivation rec {
+  pname = "openfec";
+  version = "1.4.2";
+
+  src = fetchzip {
+    url = "http://openfec.org/files/openfec_v1_4_2.tgz";
+    sha256 = "sha256:0c2lg8afr7lqpzrsi0g44a6h6s7nq4vz7yc9vm2k57ph2y6r86la";
+  };
+
+  outputs = [ "out" "dev" ];
+
+  nativeBuildInputs = [
+    cmake
+  ];
+
+  configurePhase = ''
+    mkdir build
+    cd build
+    cmake .. -DDEBUG:STRING=OFF
+  '';
```

Comment:
```suggestion
  cmakeFlags = [ "-DDEBUG:STRING=OFF" ];
```

Reply:
Thanks for the review!

Good catch. I forgot that `cmake`'s setup hook already creates `build`.